src/parsers/manifest/dash/common/indexes/utils.ts

```diff
@@ -30,3 +31,33 @@ export function getSegmentTimeRoundingError(timescale: number): number {
   return config.getCurrent().DEFAULT_MAXIMUM_TIME_ROUNDING_ERROR * timescale;
 }
 
+const supplementalCodecSeparator = /[, ]+/g;
```

Comment:
Nice, just the `indexes` directory is normally only for what we call the segment "index" here, which the way segments are announced in the MPD.

So that's probably not the right place for this util